File: Assignment_42/Program5.c
/* Write a recursive program which accept string from user and return its product of digit.
Input : 523
Output : 30  
*/

#include <stdio.h>

int Mult(int iNo)
{
   static int iMult = 1;
   int iDigit = 0;

   if(iNo < 0)
   {
    iNo = -iNo;
   }

   if(iNo != 0)
   {
        iDigit = iNo % 10;
        
        iMult = iMult * iDigit;
        iNo = iNo /10;

        Mult(iNo);
   }
       return iMult;

}

int main()
{
    int iValue = 0, iRet = 0;

    printf("Enter number :");
    scanf("%d",&iValue);

    iRet = Mult(iValue);

    printf("Product is : %d",iRet);

    return 0;
}

File: Assignment_46/Program5.java
/* Write Java Program which accept number of rows and number of columns from user and display below pattern.

Input :  iRow = 4      iCol = 5
Output:  1   2   3   4 
         5   6   7   8
         9  10  11  12
*/

import java.util.*;

class Pattern
{
    public void Pattern(int iRow,int iCol)
    {
        int i = 0, j = 0;
        int no = 1;

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                System.out.print(no+"\t");
                no++;
            }
            System.out.println();
        }
    }
}

class Program5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of Rows : ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter the number of Columns : ");
        int iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1,iValue2);
    }
}